.travis.yml

```diff
@@ -3,4 +3,4 @@ python:
   - "2.7"
   - "3.4"
 install: pip install -r requirements.txt
-script: python tests/unit-tests.py
+script: python tests/unit_test.py
```

Comment:
I renamed this file, in order to be able to also use [pytest to run the tests](https://docs.pytest.org/en/latest/goodpractices.html)